src/main/java/com/hm/achievement/listener/PlaceholderAchievementListener.java

```diff
@@ -0,0 +1,56 @@
+package com.hm.achievement.listener;
+
+import org.bukkit.entity.Player;
+
+import com.hm.achievement.AdvancedAchievements;
+import com.hm.achievement.api.AdvancedAchievementsAPI;
+import com.hm.achievement.category.MultipleAchievements;
+import com.hm.achievement.category.NormalAchievements;
+
+import me.clip.placeholderapi.external.EZPlaceholderHook;
+
+public class PlaceholderAchievementListener extends EZPlaceholderHook
+{
+	AdvancedAchievements plugin;
+	AdvancedAchievementsAPI api;
+	
+	public PlaceholderAchievementListener(AdvancedAchievements plugin, AdvancedAchievementsAPI api)
+	{
+		super(plugin, "aach");
+		this.plugin = plugin;
+		this.api = api;
+	}
+	
+	@Override
+	public String onPlaceholderRequest(Player p, String identifier)
+	{
+		for (NormalAchievements category : NormalAchievements.values())
+		{
+			String categoryName = category.toString();
+			
+			if (identifier.equalsIgnoreCase(categoryName))
+			{
+				return String.valueOf(api.getStatisticForNormalCategory(p.getUniqueId(), category));
```

Comment:
Using the API is definitely great when you're integrating from a different plugin, but you can directly use the internals of the plugin here:
`return plugin.getCacheManager().getAndIncrementStatisticAmount(category, p.getUniqueId(), 0);`

Therefore you don't actually need the `AdvancedAchievementsAPI ` object anymore and you squeeze in a little bit of performance as the API layer does some extra checks underneath the hood.